### ISSUE TITLE
Windows temp files not cleaned-up

### ISSUE THREAD

**Issue by a1880** [MiniZinc/MiniZincIDE#129]
When I start the MiniZinc IDE 2.5.2 under Windows with an empty `%TEMP%` directory, I get remaining files `mzn_*.mpc`.

Identical content of the files:

```
{
    "intermediate-solutions": true,
    "solver": "org.gecode.gecode@6.3.0"
}
```

For each of the following steps, two or three of these files are created:
1.  Starting the IDE
2.  Opening my most recent *.mzn file
3.  Running the Gecode solver

I tried and reset the Gecode solver configuration to default values, but the behaviour remained unchanged.

The files are not removed after closing the IDE.

**Comment by cyderize** [comment]
This should already be fixed in `develop` (same issue as #128) and will be in 2.5.3.

**Comment by cyderize** [comment]
This should now be in [2.5.3](https://github.com/MiniZinc/MiniZincIDE/releases/tag/2.5.3).
